### PR TITLE
Correct proptypes to reflect updated docs

### PR DESCRIPTION
**Pull request by martijnrusschen**
In 441edf887e76f51e631527d6c03c45bda673c046 we manually updated the docs, this won’t hold if we’re releasing new docs. I updated the code that’s used to generate docs. Please make sure this is correct.

### CODE REVIEW

**Comment by martijnrusschen** [comment]
@grumd @rafeememon Can you verify this is correct?

**Comment by grumd** [comment]
Seems like this is correct, see this file for reference https://github.com/Hacker0x01/react-datepicker/blob/master/src/date_input.jsx

File: src/calendar.jsx [hunk 1]
```diff
@@ -56,6 +59,7 @@ var Calendar = React.createClass({
 
   getDefaultProps () {
     return {
+      dateFormat: 'L',
```
Comment (rafeememon) [inline src/calendar.jsx]: This is passed in from the `DatePicker` component, which has a default of `'MMMM YYYY'`.  We should only define this in one place

File: src/calendar.jsx [hunk 2]
```diff
@@ -22,7 +22,10 @@ var Calendar = React.createClass({
   displayName: 'Calendar',
 
   propTypes: {
-    dateFormat: React.PropTypes.string.isRequired,
+    dateFormat: React.PropTypes.oneOfType([
+      React.PropTypes.string,
+      React.PropTypes.array
+    ]),
```
Comment (rafeememon) [inline src/calendar.jsx]: `.isRequired`

**Pull request merged**